### DISCUSSION TITLE
Unclear reason for refusal to receive Github Education

### DISCUSSION THREAD

**Discussion by ravilovrm** [community/community#133385] [GitHub Education]
### Hi everyone,

Hi everyone, 
I applied for Github Education but was rejected for the reason: Please specify school name in your next application.

On two documents - a student card with a photograph and a certificate stating that I am a student of the school, there is the full name of the school, but in Russian, since I study in Russia.

![image_2024-07-24_15-43-52](https://github.com/user-attachments/assets/cb2ad7c3-525d-41c7-8b33-ad74841f14af)

Do I need to translate documents into English somehow and submit 4 documents (2 originals and 2 copies in English) or can I just write the full name of the school in English in 1 paragraph, where it asks for the name of the school?

**Comment by ravilovrm** [comment]
up

**Comment by davevad93** [comment]
Hi @TUTBOROMA ,

The [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) says the following about translating the documentation:

- ### I need to translate my evidence. What can I do?
   - ***Ensure your billing information and identification on your academic evidence are accurate. Do you use a variation of your name or a*** ***nickname? It’s these small but important points to consider when applying that can make a much smoother process.***

   - ***We are able to translate a number of languages but if you feel that the system is unable to process your particular language then we would*** ***advise you to take a photo of the original piece of evidence next to a typed English translation, without using special characters / accent******marks.*** ***Your name on the account may need to match the translated proof as well.***

   - ***Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either.***

**Reply by ravilovrm** [reply]
https://imgur.com/a/6HoH5qM
Thank you. Do you think this will suit them? I already sent

**Reply by davevad93** [reply]
You're welcome @TUTBOROMA , I don't think it's legible enough because I think (not sure) it's a bot and not a human responsible for processing the applications, in case they reject your application for that reason I suggest you take a HD photo of each piece of your documentation and then merge them all into one photo and upload it. If I remember correctly the upload size limit is 1MB so keep that in mind.

**Reply by ravilovrm** [reply]
I can’t upload a photo, they only give you the option to upload from the camera, but there’s this blurry image, almost unreadable

**Reply by davevad93** [reply]
Ah you're right, I forgot that now there's no upload option, did they reject your application again?

**Reply by lindsaymarkward** [reply]
There are several reports of similar issues lately, so I suspect the system is not working properly. I got the same unclear reason and I have no translation or image issues. Hopefully it is fixed soon.
